src/index.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const utils = require('./utils');
```

Comment:
Instead of splitting it out can you include all the logic you've added in a single `generateScript` function in index.js?

Reply:
@lukechilds Ya it makes sense to put it all in a single file as app is small, but there should be two functions rather than a single `generateScript`. One to maintain payloads and other to generate relevant shell script.

And permission to allow changes in PR is already there.